modules/instance_template/main.tf

```diff
@@ -80,7 +87,11 @@ locals {
     id     = google_service_account.sa[0].account_id,
     email  = google_service_account.sa[0].email,
     member = google_service_account.sa[0].member
-  } : {}
+    } : {
+    id     = data.google_service_account.existing_sa[0].account_id,
+    email  = data.google_service_account.existing_sa[0].email,
+    member = data.google_service_account.existing_sa[0].member
```

Comment:
Plumbing inputs as outputs is generally discouraged but this could be okay since its reusing existing output.

Reply:
Noted.